03LessonThree/homework/git-scm.html

```diff
@@ -2,59 +2,62 @@
+  <header>
+      <nav span a>
+          <span><a href="#">Home</a></span>
+          <span><a href="#">How does it work</a></span>
+          <span><a href="#">Contact Us</a></span>
+      </nav>
+  </header>
+  <main>
+      <section>
+          <img src="http://git-scm.com//images/logo@2x.png" alt="git logo"/>
+          <h2>Welcome to Git</h2>
+          <p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. </p>
+      </section>
+      <section>
+          <section:nth-child(2)>
+              <h3>About</h3>
+              <p>The Git feature that really makes it stand apart from nearly every other SCM out there is its branching model. Git allows and encourages you to have multiple local branches that can be entirely independent of each other. The creation, merging, and deletion of those lines of development takes seconds. This means that you can do things like:</p>
+              <ul>
+                  <li>Frictionless Context Switching. Create a branch to try out an idea, commit a few times, switch back to where you branched from, apply a patch, switch back to where you are experimenting, and merge it in.</li>
+                  <li>Role-Based Codelines. Have a branch that always contains only what goes to production, another that you merge work into for testing, and several smaller ones for day to day work.</li>
+                  <li>Feature Based Workflow. Create new branches for each new feature you're working on so you can seamlessly switch back and forth between them, then delete each branch when that feature gets merged into your main line.</li>
+                  <li>Disposable Experimentation. Create a branch to experiment in, realize it's not going to work, and just delete it - abandoning the work—with nobody else ever seeing it (even if you've pushed other branches in the meantime).</li>
+              </ul>
+          </section:nth-child(2)>
+          <section:nth-child(2)>
+              <h3>Community</h3>
+              <p>Questions or comments for the Git community can be sent to the mailing list by using the email address git@vger.kernel.org. Bug reports should be sent to this mailing list.</p>
+              <p>You do not need to subscribe: you will be Cc'd in replies. Please keep the Cc list intact when replying (use "Reply to all"). Greylisting may delay your first post for a few hours. Note also that the list only accepts plain-text email; please disable HTML in your outgoing messages.</p>
+              <p>The archive can be found on Gmane. Click here to subscribe. If you have trouble sending to the list with your regular mail reader, you may also post via gmane. There is also Git user mailing list on google Groups which is a nice place for beginners to ask about anything.</p>
+          </section:nth-child(2)>
+      </section>
+      <aside>
+          <section>

```

Comment:
This should have just been a regular `<div>`.
Good job otherwise, just be more careful next time. The CSS is adding styles to `section` elements and you inadvertently caused your page to not render correctly.